shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/ExpressionBuilder.java

```diff
@@ -50,13 +52,7 @@ public OrPredicateSegment extractAndPredicates() {
                 result.getAndPredicates().addAll(leftBuilder.extractAndPredicates().getAndPredicates());
                 result.getAndPredicates().addAll(rightBuilder.extractAndPredicates().getAndPredicates());
             } else if (logicalOperator.isPresent() && LogicalOperator.AND == logicalOperator.get()) {
-                ExpressionBuilder leftBuilder = new ExpressionBuilder(((BinaryOperationExpression) expression).getLeft());
-                ExpressionBuilder rightBuilder = new ExpressionBuilder(((BinaryOperationExpression) expression).getRight());
-                for (AndPredicate eachLeft : leftBuilder.extractAndPredicates().getAndPredicates()) {
-                    for (AndPredicate eachRight : rightBuilder.extractAndPredicates().getAndPredicates()) {
-                        result.getAndPredicates().add(createAndPredicate(eachLeft, eachRight));
-                    }
-                }
+                processAndExpression(result);
```

Comment:
@vince7839 Maybe `result.getAndPredicates().addAll(createAndPredicates())` is better.

Reply:
Do you mean changing the method of "processAndExpression()" to "createAndPredicates()"?

Reply:
> Do you mean changing the method of "processAndExpression()" to "createAndPredicates()"?

@vince7839 Yes.

Reply:
Okay,I'll do it and add UT